gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -423,6 +439,20 @@ private void registerRouteRegistrationHandler(Handler<Void> readyHandler) {
         readyHandler.handle(null);
     }
 
+    /**
+     * Update all registered router's pool size multiplier
+     *
+     * @param readyHandler - the ready handler
+     */
+    private void registerRouteMultiplierChangeHandler(Handler<Void> readyHandler) {
+        vertx.eventBus().consumer(Router.ROUTE_MULTIPLIER_ADRESS, (Handler<Message<String>>) event -> {
+            log.warn("Update router's pool size multiplier: {}", (event.body() == null ? "<null>" : event.body()));
+            routeMultiplier = Integer.parseInt(event.body());
```

Comment:
I think we should catch exceptions here like _NumberFormatException_. What happens when we get a non-parsable String?

Reply:
OK, will add